docs/input/opc-ua-input.md

```
@@ -292,3 +295,63 @@ input:
+
+**Risks of incorrect configuration**:
+- **Too small**: May cause notification loss during high-frequency periods
+- **Too large**: Excessive memory usage that could impact system performance 
+
+```yaml
+input:
+  opcua:
+    queueSize: 50  # Only increase if you experience notification loss
+```
+
+**Sampling Interval**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)